setup.py

```diff
@@ -48,7 +60,12 @@ def write_version_to_file(version, target_file):
         author_email='shaoshuaics@gmail.com',
         license='Apache License 2.0',
         packages=find_packages(exclude=['tools', 'data', 'output']),
-        cmdclass={'build_ext': BuildExtension},
+        cmdclass={
+            'build_ext': BuildExtension,
+            'install': PostInstallation,
```

Comment:
In that case, you may comment out this line. Then it will not install kornia by default. @sshaoshuai

Reply:
Got it, thank you!